spec/index.bs

```diff
@@ -8952,14 +8956,6 @@ GPUQueue includes GPUObjectBase;
             1. Let |dataBytes| be [=get a copy of the buffer source|a copy of the bytes held by the buffer source=] |data|.
             1. Let |dataByteSize| be the number of bytes in |dataBytes|.
             1. Let |textureDesc| be |destination|.{{GPUImageCopyTexture/texture}}.{{GPUTexture/[[descriptor]]}}.
-            1. If any of the following conditions are unsatisfied,
```

Comment:
Note: moving this validation from the content timeline to the device timeline is something we'll likely discuss next meeting (it is very likely that it will be agreed to) https://github.com/gpuweb/gpuweb/issues/2680

Reply:
Okay. I'll land this as-is, and then whoever makes the edit to move the validation to the device timeline can move it along with all its new neighbors.